### PR TITLE
BaSP-TG-58: Setup react router

### PR DESCRIPTION
**Pull request by CristianBucci**
Install react router library, implement react router in the layout and in the components that are performing redirects

### CODE REVIEW

File: src/Components/Admins/Form/index.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 
-const Form = () => {
+const Form = (props) => {
+  const params = useParams();
+  const id = params.Id ? params.Id : '';
+  console.log(params);
```
Comment (matescarabino) [inline src/Components/Admins/Form/index.js:7]: Remove console.logs before merge

File: src/Components/TimeSheets/index.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 import Modal from './Modal/index';
 import styles from './timeSheets.module.css';
 
-const TimeSheets = () => {
+const TimeSheets = (props) => {
   const [timeSheets, setTimesheet] = useState([]);
   const [showModal, setShowModal] = useState(false);
   const [timeSheetId, setTimeSheetId] = useState(undefined);
+  const params = useParams();
+  console.log(params);
```
Comment (matescarabino) [inline src/Components/TimeSheets/index.js:12]: Same here

**Review by matescarabino — CHANGES_REQUESTED** [review]
Everything looks fine, only remove the console.logs left

File: src/Components/Admins/Form/index.js (hunk 1 above)
Comment (CristianBucci) [reply src/Components/Admins/Form/index.js]: Done!

File: src/Components/TimeSheets/index.js (hunk 2 above)
Comment (CristianBucci) [reply src/Components/TimeSheets/index.js]: Done!

**Review by nicolasblasco — APPROVED** [review]

**Review by matescarabino — APPROVED** [review]

**Pull request merged**